public/app/features/explore/Logs/LogsTableWrap.tsx

```diff
@@ -35,11 +35,23 @@ interface Props extends Themeable2 {
   datasourceType?: string;
 }
 
-export type fieldNameMeta = {
+type activeFieldMeta = {
```

Comment:
I think based on our styleguide we should capitalise all of those type names at one point: https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#types-and-interfaces 

Not necessary in this PR.